java/src/jmri/jmrix/loconet/LocoNetSystemConnectionMemo.java

```diff
@@ -129,6 +129,7 @@ public void setLncvDevicesManager(LncvDevicesManager lncvdm) {
 
     protected boolean mTurnoutNoRetry = false;
     protected boolean mTurnoutExtraSpace = false;
+    protected boolean mInterrogate = true;
```

Comment:
This defaults to false in LnPortController.  Should they be consistent?

Reply:
Thanks, I will check that, true should be the default everywhere, including setting a new connection. And get back to you.

Reply:
Done